lib/qiita/markdown/filters/external_link.rb

```diff
@@ -0,0 +1,35 @@
+module Qiita
+  module Markdown
+    module Filters
+      class ExternalLink < HTML::Pipeline::Filter
+        def call
+          return doc unless hostname
```

Comment:
Since the pipeline is pluggable, people who don't need the `ExternalLink` filter should remove it from the pipeline by themselves. That's the convention of `HTML::Pipeline` framework, IMO.